### DISCUSSION TITLE
Prevent scrollbar from showing when resizing window

### DISCUSSION THREAD

**Discussion by Codebreaker101** [quasarframework/quasar#10913] [General - Components / Directives / etc] [answered]
You can see it in action by going to https://quasar.dev/layout-builder and resize the window vertically.

I think this is caused by `Screen` plugin by setting `min-size` inline style on a `div.q-layout` trough a debounce function and that delay is causing it to show. I can also see my content twitching (canvas) since I'm calculating stuff based on height.

How to prevent the scroll bar from showing in q-layout?

**Comment by pdanpdan** [comment, marked as answer]
Change the denounce time
https://quasar.dev/options/screen-plugin#configuration

**Reply by Codebreaker101** [reply]
Setting `$q.screen.setDebounce(0);` does indeed solve the issue
